### DISCUSSION TITLE
10.9 and HW acceleration on Apple Silicon mac

### DISCUSSION THREAD

**Discussion by ghost** [jellyfin/jellyfin#11137] [Q&A] [answered]
I recently learned that 10.9 is finally coming and I'm preparing to migrate my server from my poor Raspberrypie to M1 mac.
Previous versions of Jellyfin did't support arm64 and some HW accelerations on mac, and I previously heard arm64 support is coming to mac in 10.9, but I'm curious about HW acceleration on mac.
According to [this thread](https://github.com/jellyfin/jellyfin-ffmpeg/issues/339), it seems like there has been a major breakthrough at Jellyfin-ffmpeg. Will 10.9 come with improved HW acceleration on mac? If so, will I need to do additional configuration or installation after installation, or does jellyfin-ffmpeg comes with macOS Jellyfin server installer by default?

**Comment by gnattu** [comment, marked as answer]
The preliminary document is available here if you are interested: https://github.com/gnattu/jellyfin.org/blob/apple-doc/docs/general/administration/hardware-acceleration/apple.md

We can expect full hardware acceleration on Apple Silicon Macs running macOS 12+, and the `jellyfin-ffmpeg` should be bundled with https://github.com/jellyfin/jellyfin-server-macos, you will also be able to download it manually once released.

The configuration on macOS should be much simpler, and you only have to set it up in the Jellyfin dashboard, as the system-related functions work out of the box.

**Reply by ghost** [reply]
Thanks for fast and detailed response and hard development work!
I'm exited to switch from my slow Raspberry pi.
